### PR TITLE
[dv] Add automated alert_test to all IPs with new tl_int alert

### PR DESCRIPTION
**Pull request by cindychip**
This PR adds alert_test to {IP}_sim_cfg.hjson and {IP}_testplan
This PR also adds interrupt test to SPI_HOST.

Signed-off-by: Cindy Chen <chencindy@opentitan.org>

### CODE REVIEW

File: hw/ip/spi_host/dv/spi_host_sim_cfg.hjson [hunk 1]
```diff
@@ -30,6 +30,8 @@
                 // Common CIP test lists
                 "{proj_root}/hw/dv/tools/dvsim/tests/csr_tests.hjson",
                 "{proj_root}/hw/dv/tools/dvsim/tests/mem_tests.hjson",
+                "{proj_root}/hw/dv/tools/dvsim/tests/alert_test.hjson",
+                "{proj_root}/hw/dv/tools/dvsim/tests/intr_test.hjson",
```
Comment (cindychip) [inline hw/ip/spi_host/dv/spi_host_sim_cfg.hjson:34]: @tunghoang290780 looks like the interrupt auto test is missing here. Please let me know if you have any concerns.

**Review by udinator — APPROVED** [review]
lgtm!

**Review by matutem — APPROVED** [review]
LGTM

**Review by weicaiyang — APPROVED** [review]
LGTM, thanks

**Pull request merged**